mibitracker/request_helpers.py

```diff
@@ -557,14 +557,16 @@ def get_channel_data(self, image_id, channel_name):
         buf.seek(0)
         return skio.imread(buf)
 
-    def create_imageset(self, image_ids, imageset_name,
+    def create_imageset(self, image_ids, imageset_name, project_id,
                         imageset_description=None):
         """Creates a new imageset with the specified images.
 
         Args:
             image_ids: A list of ints of ids of the images in MibiTracker
                 corresponding to the images to be added to the new imageset.
             imageset_name: A string name for the new imageset.
+            project_id: An integer number for identifying the project to which
+                the images belong.
```

Comment:
The project id specifies where the _imageset_ belongs. The images must be either in that project, or in other projects to which the user has access and have sharing enabled. So the docstring should be something like "An integer id specifying the project in which to create the imageset. Note that all images in the set must be either in this project or shared with the user."

Reply:
Done.